src/buffer/out/textBuffer.cpp

```diff
@@ -1189,6 +1189,8 @@ void TextBuffer::ClearScrollback(const til::CoordType start, const til::CoordTyp
         return;
     }
 
+    ClearMarksInRange(til::point{ 0, 0 }, til::point{ _width, height });

```

Comment:
```suggestion
    ClearMarksInRange(til::point{ 0, start }, til::point{ _width, start + height });
```

wait, don't you need this?

Reply:
wait, if we `_decommit` if height is 0, do we also want to wipe out all marks?

Reply:
Oh yeah, it needs to be moved above the `if`. And I didn't even notice that the point was different. Clearly, I should review this on Monday again. 😄